yb-voyager/cmd/analyzeSchema.go

```diff
@@ -134,6 +134,7 @@ var (
 	parserIssueDetector = queryissue.NewParserIssueDetector()
 	multiRegex          = regexp.MustCompile(`([a-zA-Z0-9_\.]+[,|;])`)
 	dollarQuoteRegex    = regexp.MustCompile(`(\$.*\$)`)
+	sqlBodyBeginRegex   = re("BEGIN", "ATOMIC")
```

Comment:
Should we put a `^` in the regex to ensure that `BEGIN ATOMIC` is at the start of line?
Do we need to ensure that?

Reply:
I don't think we need to ensure that as it could be anywhere  not necessarily in starting of the line e.g.
```
CREATE FUNCTION public.asterisksdfsf(n integer) RETURNS SETOF text
    LANGUAGE sql IMMUTABLE STRICT PARALLEL SAFE BEGIN ATOMIC
 SELECT repeat('*'::text, g.g) AS repeat
    FROM generate_series(1, asterisksdfsf.n) g(g);
END;

```